### ISSUE TITLE
Regarding Duration Handling in Activity

### ISSUE THREAD

**Issue by dibeshdas151995** [uber/cadence#3430]
I have been trying to find out someway some way to provide some sleep to the activities.
let's say there are three activities, Activity A,B and C. When A is executed successfully, then I want to provide a halt of 30 minutes. Then start the process for B and after completion of B, C will start executing and in between the Activity of C, I want to provide some sleep functionality in C. And lastly C will be completed. 

In short, I want to provide sleep/halt functionality in Activity level.

Is it possible anyway? Please suggest something.

**Comment by vancexu** [comment]
closed as duplicate https://github.com/uber/cadence/issues/3431
